src/views/Admin/DogAdmin.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { addDog } from '../../services/dogs';
+import { useHistory } from 'react-router-dom';
+import DogForm from '../../components/DogForm/DogForm';
+import './DogAdmin.css';
+
+export default function DogAdmin() {
+  const [name, setName] = useState('');
+  const [breed, setBreed] = useState('');
+  const [image, setImage] = useState('');
+  const [bio, setBio] = useState('');
+  const [message, setMessage] = useState('');
+
+  const history = useHistory();
+
+  const handleSubmit = async (e) => {
+    try {
+      e.preventDefault();
+      const data = await addDog(name, breed, image, bio);
+      setMessage('Success! Redirecting...');
+      setTimeout(() => {
+        history.push(`/dogs/${data[0].id}`);
+      }, 2500);
+    } catch {
+      setMessage('Oops! Something went wrong!');
+    }
+  };
+  return (
+    <div>
+      <p className="message">{message}</p>
```

Comment:
If you set the initial message state to `null` and use `{message && message}` you can avoid having an empty `<p>` on the page

Reply:
oooo! i updated it!